src/directive.ts

```diff
@@ -132,7 +165,7 @@ function buildApiSchema(validationSchema: string[] | undefined, argValue: ConstV
   return `.${schemaApi}(${schemaApiArgs.join(', ')})`;
 }
 
-function buildApiFromDirectiveArguments(config: FormattedDirectiveArguments, args: ReadonlyArray<ConstArgumentNode>): string {
+function buildApiFromDirectiveArguments(config: FormattedDirectiveArguments, args: ReadonlyArray<ConstArgumentNode>): string[] {
```

Comment:
I would like to be happy to reduce destructive changes.
Could you define a call to `_buildApiFromDirectiveArguments(): string[]` inside `buildApiFromDirectiveArguments: string`?

Also changes `buildApiForValibot` using `_buildApiFromDirectiveArguments(): string[]`?

Reply:
Certainly! That would reduce the impact on existing features. I'll fix it!

Reply:
Corrected here: https://github.com/Code-Hex/graphql-codegen-typescript-validation-schema/pull/667/commits/b9cb35b5b4dc640931910fab5ffaa087a42c54c6
This change has eliminated the impact on existing tests 🙌🏻